Authenticator/Source/TokenListViewController.swift

```diff
@@ -139,10 +143,8 @@ extension TokenListViewController {
     }
 
     override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCellWithIdentifier(NSStringFromClass(TokenRowCell.self), forIndexPath: indexPath)
-        if let cell = cell as? TokenRowCell {
-            updateCell(cell, forRowAtIndexPath: indexPath)
-        }
+        let cell = tableView.dequeueReusableCellWithClass(TokenRowCell.self)
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 118 characters (line_length)